scripts/devel/build_and_install.py

```diff
@@ -16,16 +16,53 @@
 
 import argparse
 import os
+import subprocess
 import sys
 
 from ros_buildfarm.argument import add_argument_build_tool
 from ros_buildfarm.argument import add_argument_build_tool_args
+from ros_buildfarm.argument import add_argument_ros_version
+from ros_buildfarm.argument import add_argument_run_abichecker
 from ros_buildfarm.common import Scope
 from ros_buildfarm.workspace import call_build_tool
 from ros_buildfarm.workspace import clean_workspace
 from ros_buildfarm.workspace import ensure_workspace_exists
 
 
+def call_abi_checker(workspace_root, ros_version, env):
+    # import the module only if the function is being called to reduce the
+    # number of mandatory dependencies
+    from catkin_pkg.packages import find_packages
+
+    # TODO: pkgs detection, code based on create_devel_task_generator.py
```

Comment:
If this is already worth a TODO why not deduplicate the logic?

Reply:
not really. Waiting travis to pass through the changes for this (and for fixing the CI) before push them here.

Reply:
Refactored here 9d8b0c5

Reply:
I refactored the new function a bit in c8673893c435541640ba7f36324d1a394b899a2c.